qsv/test/org/qcmg/qsv/util/TestUtil.java

```diff
@@ -273,13 +273,14 @@ public static File createHiseqBamFile(final String inputFileName, PairGroup pg,
         
 		SAMFileWriterFactory factory = new SAMFileWriterFactory();
 		factory.setCreateIndex(true);
-		SAMFileWriter writer = factory.makeBAMWriter(header, false, new File(inputFileName));
+		try (SAMFileWriter writer = factory.makeBAMWriter(header, false, new File(inputFileName));) {
```

Comment:
create the SamReader in the `try () { }`  too while we're at it? Then no calls to close()

indentation is bit screwy in this block